components/Modals/BuyingGuideModal.tsx

```diff
@@ -195,7 +194,13 @@ export default function BuyingGuide({
             buyWithProvider();
             if (provider !== "Ratio") setIsProviderStepDone(true);
           }}
-          done={provider === "Ratio" ? buyRatioDone : isProviderStepDone}
+          done={
+            provider === "Ratio"
+              ? buyRatioDone
+              : isProviderStepDone ||
+                (balance &&
+                  BigNumber.from(balance.value).gte(utils.parseEther("1000.0")))
```

Comment:
should be dynamic? 

Also due to how ethereum works I think we'll often get slightly smaller than exactly 1000 numbers. Let's do the number it's checking -1%? So `inputNumber * 0.99` ?

Reply:
ok, adding